replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/persistence/ReplicatorConfigManagerImpl.java

```diff
@@ -43,4 +45,16 @@ public void save(ReplicatorConfig replicatorConfig) {
   public void remove(String id) {
     persistentStore.delete(ReplicatorConfigImpl.class, id);
   }
+
+  @Override
+  public boolean configExists(String configId) {
+    try {
+      persistentStore.get(ReplicatorConfigImpl.class, configId);
+    } catch (ReplicationPersistenceException e) {
+      return true;
+    } catch (NotFoundException e) {
+      return false;
+    }
+    return true;
+  }
```

Comment:
✏️ could just use the get(id) method and if it throws the NotFoundException you know the config doesn't exist. Also, it seems a little misleading to return true when we get a ReplicationPersistenceException.

Reply:
I would agree that we shouldn't return true on a ReplicationPersistenceException

Reply:
I changed to return false for now, but I think we will want to throw some sort of error and handle on the front end if we cannot access the persistence store.

Reply:
I think that is exactly right. We want to throw an exception up if there is an error instead of returning true/false.